### PR TITLE
Make new Custom Taxonomies Hierarchical by default

### LINKED ISSUE #3846: Make default for new Custom Taxonomies be Hierarchical
Currently new taxonomies are tag-like, we should use category-like for the default instead.

### PR DESCRIPTION
**Pull request by JoryHogeveen**
Fixes #3846

### CODE REVIEW

File: classes/PodsAdmin.php [hunk 1]
```diff
@@ -1295,7 +1295,7 @@ public function admin_setup_edit_options ( $pod ) {
                     'label' => __( 'Hierarchical', 'pods' ),
                     'help' => __( 'help', 'pods' ),
                     'type' => 'boolean',
-                    'default' => false,
```
Comment (sc0ttkclark) [inline classes/PodsAdmin.php]: Not just here but also in the create new pod function in PodsAPI.

**Comment by sc0ttkclark** [comment]
Needs some adjustment to move forward.

File: classes/PodsAdmin.php (hunk 1 above)
Comment (JoryHogeveen) [inline classes/PodsAdmin.php]: I've put it in save_pod(). Please test this as well!

**Comment by JoryHogeveen** [comment]
@sc0ttkclark should that be in save pod? I wrote it in add_pod at first but then noticed that all saves pass through the save_pod so thought this would be a better location.

**Comment by sc0ttkclark** [comment]
In the future we will be moving some of the defaults from add_pod to save_pod, we can move it there at that time. I just didn't want it to get left behind with the defaults that we use in add_pod explicitly.

**Pull request merged**